eth/gasprice/feehistory.go

```diff
@@ -54,10 +52,15 @@ type blockFees struct {
 	block       *types.Block // only set if reward percentiles are requested
 	receipts    types.Receipts
 	// filled by processBlock
+	processedFees
```

Comment:
Can we separate the "result" out from the blockFees struct? It's not so nice to have the embedded struct.